core/src/main/java/Scenes/Lobby.java

```diff
@@ -16,78 +18,349 @@
+  // References to the buttons that are currently pressed.
+  private TextButton pressedCategory;
+  private TextButton pressedDifficulty;
+  private TextButton pressedLength;
 
   /**
    * Refer to MainMenu.java for comments regarding each section. Lobby should operate in the same
    * way with changes to the textures/sprites that must be drawn to the screen.
    */
   public Lobby(FilePlayMain mainGame) {
     super(mainGame);
-    // Creates a button using the given texture at location (20, 650) of the native resolution 480
-    // by 800.
-    backButton = new ButtonActor(new Texture(Gdx.files.internal("back_button.png")), 20, 650);
-    // Creates a button using the given texture at location (320, 650) of the native resolution 480
-    // by 800.
-    playButton = new ButtonActor(new Texture(Gdx.files.internal("next_button.png")), 320, 650);
-    lobbyScreenText = new BitmapFont();
-    lobbyScreenText.setColor(Color.YELLOW);
-    playText = new BitmapFont();
-    playText.setColor(Color.YELLOW);
-    lobbyMockUp = new Texture(Gdx.files.internal("lobby_mockup.png"));
+    // Creates GlyphLayout to get width for centering text in the banner.
+    bannerTextGlyphLayout = new GlyphLayout(bannerText, "Lobby");
+    // Calculate the center for the text to be drawn in the banner.
+    glyphCenterX = ((int) WORLD_WIDTH - (int) bannerTextGlyphLayout.width) / 2;
+    labelFont = generateNewFont("BROADSolid.ttf", 36, Color.WHITE);
+    createButtons();
     addAllListeners();
     addAllActors();
   }
 
   @Override
   public void render(float delta) {
     super.render(delta);
-    spriteBatch.setProjectionMatrix(camera.combined);
+    stage.act(delta);
+    stage.draw();
     spriteBatch.begin();
-    // Draws the text "Lobby Screen (To be implemented)" at the location (20, 750) of the native
-    // resolution 480 by 800.
-    lobbyScreenText.draw(spriteBatch, "Lobby Screen (To be implemented)", 20, 750);
-    // Draws the text "Play" at the location (400, 750) of the native resolution 480 by 800.
-    playText.draw(spriteBatch, "Play", 400, 750);
-    // Draws a sprite using the given texture at a location of (65, 300) of the native resolution.
-    spriteBatch.draw(new Sprite(lobbyMockUp), 65, 300);
+    // Draws the text "Lobby" in the center of the banner.
+    bannerText.draw(spriteBatch, bannerTextGlyphLayout, glyphCenterX, 770);
+    // Draws all the labels for each section.
+    labelFont.draw(spriteBatch, "Categories", LABELX, 690);
+    labelFont.draw(spriteBatch, "Difficulty", LABELX, 470);
+    labelFont.draw(spriteBatch, "Game Length", LABELX, 300);
     spriteBatch.end();
   }
 
   /**
+   * Creates all of the buttons that will be drawn to the screen.
+   */
+  @Override
+  protected void createButtons() {
+    backButton = new ButtonActor(new Texture(Gdx.files.internal("black-back-arrow.png")), 0, 735);
+    TextButtonStyle style;
+    style = setStyle("nano yellow", "nano yellow");
+    playButton = new TextButton("Play", style);
```

Comment:
Instead of assigning style with a new TextButtonStyle before each button instantiation, you can probably place the setStyle() call directly in the argument list.